draft-demarco-oauth-status-attestations.md

```diff
@@ -133,27 +133,23 @@ validity of the Digital Credentials, including their revocation status,
 before accepting them.
 
 Wallet Instance:
-: The digital Wallet in control of a User, also known as Wallet or Holder.
```

Comment:
I dind't want to get in conflict about other specifications with different assumptions about what the holder, technically, is.